Card/src/main/java/com/braintreepayments/api/BaseCard.java

```diff
@@ -376,7 +377,8 @@ JSONObject buildJSON() throws JSONException {
     }
 
     @Override
-    String getApiPath() {
+    @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
```

Comment:
Curious - when we do we need `@suppress` annotations, like [here](https://github.com/braintree/braintree_android/blob/fbf630e60e689ee0f1bd99f4d2969bcdcb17a921/BraintreeCore/src/main/java/com/braintreepayments/api/Configuration.kt#L108)?

Reply:
We use `@suppress` to keep the Kotlin docs from being generated. I can add some here.